### DISCUSSION TITLE
Retirer le numero de version et le mot English

### DISCUSSION THREAD

**Discussion by anistouati** [MTESSDev/FRW#302] [Annonces]
Bonjour, 

J'ai un retour du PO qui suggère le retrait du mot Engish en haut et le numéro de la version en bas de la page du formulaire FRW.

Merci!

![image](https://github.com/user-attachments/assets/6590c5a3-fea7-4e75-ac7f-75ee9639f8b1)

**Comment by Hutson48** [comment]
Bonjour,

En ce qui concerne l'affichage de la langue, il est effectivement possible de configurer le formulaire en mode unilingue, comme indiqué dans notre documentation: https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=personnalisationBase

Quant au numéro de version, ce dernier n'est malheureusement pas modifiable pour le moment. Nous vous tiendrons informé si des changements à ce sujet sont apportés.

N’hésitez pas à revenir vers nous pour toute autre question.

**Reply by anistouati** [reply]
Merci pour votre réactivité, je transfert le message.
